review_practice/api-server/database/models/index.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import path from 'path';
+import Sequelize from 'sequelize';
+import configs from '../../config/database.js';
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = configs[env];
+
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+    sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+    sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs.readdirSync(__dirname)
+    .filter(file => {
+        return file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js';
+    })
+    .forEach(file => {
+        const model = sequelize['import'](path.join(__dirname, file));
+        db[model.name] = model;
+    });
+
+Object.keys(db).forEach(modelName => {
+    if (db[modelName].associate) {
+        db[modelName].associate(db);
+    }
+});
```

Comment:
일단 Object.keys 대신에 values를 사용해보면 어떨까요?
```suggestion
Object.values(db)
      .filter(model => model.associate)
      .forEach(model => model.associate(db));
```
그리고 다음과 같이 `optional channing`을 이용하는 방법도 있답니다!
```suggestion
Object.values(db).forEach(model => model.associate?.(db));
```